### ISSUE TITLE
No source distribution available

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tenei17/macfuse#4]
```
I'm looking into the viability of writing a MacPorts Portfile, but without a 
source distribution 
available this is going to be a problem.
```

Original issue reported on code.google.com by `kball...@gmail.com` on 11 Jan 2007 at 10:08

**Comment by GoogleCodeExporter** [comment]
```
Then what did I just download via the SVN link in the "Source" tab at the top 
of the project page?
```

Original comment by `steve.p...@gmail.com` on 11 Jan 2007 at 10:32

**Comment by GoogleCodeExporter** [comment]
```
steve, that's called checking out a repository. Different than a source 
distribution. What you got was the 
current bleeding edge development version, and it required using subversion. 
What I want is a tarball that 
contains the exact source used to build the current binary distribution, the 
idea being I can download that 
(without having subversion installed) and compile/install it myself.

Caveat: the current binary distribution also includes glib/gettext/pkg-config 
but I don't want those in the 
source tarball. I can provide those libraries myself. Same with sshfs - I can 
provide that myself, though the 
patch in the repo is necessary.

Basically, I want an export of the repository as it looked when the binary 
distribution was built.
```

Original comment by `kball...@gmail.com` on 11 Jan 2007 at 10:35

**Comment by GoogleCodeExporter** [comment]
```
You can ask svn for a specific revision and that will stay constant until the 
end of time. Is it really too hard to 
install subversion and make your own snapshots?
```

Original comment by `its.fl...@gmail.com` on 12 Jan 2007 at 7:21

**Comment by GoogleCodeExporter** [comment]
```
I see your point, particularly if you are doing it for the MacPorts system 
where you presumably want to be able to 
download a known file from a known place and apply known sets of patches to it. 
 Apologies for jumping the gun 
on that one.  Given this is an infant project, I suspect when a "stable" or at 
least seems-to-be-pretty-darn-stable 
[smile] release comes out, that will be part of the downloadable filesets.
```

Original comment by `steve.p...@gmail.com` on 12 Jan 2007 at 3:47

**Comment by GoogleCodeExporter** [comment]
```
Kevin, I thought Macports was on the way to require svn to be built. Moreover, 
there 
is a cvs module in macports that permits checkouting source from a cvs 
repository, 
and if there is no for svn, then it should be on the todo list.
```

Original comment by `hai...@gmail.com` on 12 Jan 2007 at 7:01

**Comment by GoogleCodeExporter** [comment]
```
hainry - I haven't heard anything about MacPorts ever requiring svn.

Just like the cvs module, it also supports svn, but that requires the user to 
have subversion installed. I'd rather 
not force the user to install subversion when MacFUSE itself has nothing to do 
with subversion.
```

Original comment by `kball...@gmail.com` on 13 Jan 2007 at 12:00

**Comment by GoogleCodeExporter** [comment]
```
And why not use the tagged version of the repository ?
http://macfuse.googlecode.com/svn/tags/

That is a source distribution right ?
```

Original comment by `Christia...@gmail.com` on 25 Jan 2007 at 4:00

**Comment by GoogleCodeExporter** [comment]
```
For now, we can only do tagged versions of the repository. The web site doesn't 
make such versions 
automatically available as tarballs etc.
```

Original comment by `si...@gmail.com` on 29 Jan 2007 at 4:33
- Changed state: **WontFix**

**Comment by GoogleCodeExporter** [comment]
```
asingh: you provide binary releases. Why can't you just export a tag, tarball 
it, and put it up for download next to 
the binary release?
```

Original comment by `kball...@gmail.com` on 29 Jan 2007 at 9:57
